### PR TITLE
Add Kafka authorization tutorial

### PR DESCRIPTION
**Pull request by tsandall**
Signed-off-by: Torin Sandall <torinsandall@gmail.com>

### CODE REVIEW

File: docs/book/kafka-authorization.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+
+Lastly, run `kafka-console-producer` to simulate a service that should **not**
+have access to _high fanout_ topics.
+
+```bash
+docker run --rm --network opakafkatutorial_default \
+    openpolicyagent/demo-kafka:1.0 \
+    bash -c 'echo "{\"user\": \"alice\", \"button\": \"bogus\"}" | kafka-console-producer --topic click-stream --broker-list kafka:9093 -producer.config /etc/kafka/secrets/anon_producer.ssl.config'
+```
+
```
Comment (ashutosh-narkar) [inline docs/book/kafka-authorization.md]: Can we add some part of the output of this command:
```
[2018-04-26 07:56:38,264] ERROR Error when sending message to topic click-stream with key: null, value: 36 bytes with error: (org.apache.kafka.clients.producer.internals.ErrorLoggingCallback)
org.apache.kafka.common.errors.TopicAuthorizationException: Not authorized to access topics: [click-stream]```

File: docs/book/kafka-authorization.md [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+## Goals
+
+This tutorial shows how to enforce fine-grained access control over Kafka
+topics. In this tutorial you will use OPA to define and enforce an
+authorization policy stating:
+
+* Consumers of topics containing Personally Identifiable Information (PII) must be whitelisted.
+* Producers to topics with _high fanout_ must be whitelisted.
+
+In addition, this tutorial shows how to break up a policy with a small helper
```
Comment (ashutosh-narkar) [inline docs/book/kafka-authorization.md]: NIT: _with small helper rules_

File: docs/book/kafka-authorization.md (hunk 1 above)
Comment (tsandall) [reply docs/book/kafka-authorization.md]: Done.

File: docs/book/kafka-authorization.md (hunk 2 above)
Comment (tsandall) [reply docs/book/kafka-authorization.md]: Done.

**Pull request merged**